src/fairseq2/data/vocabulary_info.py

```diff
@@ -10,6 +10,8 @@
 
 @dataclass
 class VocabularyInfo:
+    """Describes the vocabulary used by a :py:class:`fairseq2.data.text.TextTokenizer`"""
```

Comment:
True, but do you have a better class to use in place of `TextTokenizer` ? I don't think we have a more generic `Tokenizer` class.

Reply:
You can't have a more generic `Tokenizer` class since they have different input modalities and their APIs are different. However `TextTokenizer` and `VocabularyInfo` are loosely coupled. You can check out `UnitTokenizer` [here](https://github.com/facebookresearch/seamless_communication/blob/main/src/seamless_communication/models/unity/unit_tokenizer.py). It follows similar idioms as `TextTokenizer`, but it is essentially a different type with different input types.

Reply:
I've removed the reference to a specific class, in favor of the generic "tokenizer" term